sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c

```diff
@@ -299,6 +300,10 @@ void calc_fast9_lukas_kanade(struct opticflow_t *opticflow, struct opticflow_sta
   float diff_flow_x = 0;
   float diff_flow_y = 0;
 
+  // Flow Derotation TODO:
```

Comment:
Yeah the TODO,,,, was about the unknown impact, for me it is fine, all tests I've done were OK. Maybe K. McGuire has still something to add?